rapla-parser/src/structs.rs

```diff
@@ -21,4 +21,5 @@ pub struct Event {
     pub location: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub organizer: Option<String>,
+    pub description: Option<String>,
```

Comment:
Can you add `#[serde(skip_serializing_if = "Option::is_none")]` and extend the table in README.md under "Usage as JSON API"?